src/components/TextField/README.md

```diff
@@ -733,7 +733,11 @@ class SeparateValidationErrorExample extends React.Component {
             <InlineError message={errorMessage} fieldID={textFieldID} />
           </div>
         </Stack.Item>
-        <Button icon={DeleteMinor} />
+        <Button
+          icon={DeleteMinor}
+          accessibilityLabel="Remove item"
+          labelHidden
```

Comment:
Does `Button` have a `labelHidden` prop?

Reply:
Nope my bad. This line should be removed.
```suggestion
```